### DISCUSSION TITLE
Z-Wave Silicon Labs 800 Series module (ZGM230S) Firmware

### DISCUSSION THREAD

**Discussion by vignez2991** [zwave-js/zwave-js#6665] [General] [answered]
Where can I get the Updated Firmware for Z-Wave Silicon Labs 800 Series module

**Comment by AlCalzone** [comment, marked as answer]
https://github.com/SiliconLabs/gecko_sdk provide the official pre-built firmwares.

For the current release you'll find them in the https://github.com/SiliconLabs/gecko_sdk/tree/gsdk_4.4/protocol/z-wave/Apps/bin/gbl directory under `zwave_ncp_serial_api_controller_....gbl`. 
You'll have to figure out which of those is for your device, since there are multiple ones for the 800 series. DO NOT use the _v255 files.

https://github.com/kpine/zwave-js-server-docker/wiki/700-series-Controller-Firmware-Updates-(Linux)#firmware-downloads might help too.
